API/NoDocs/Microsoft.AspNetCore.Telemetry/HttpLoggingServiceExtensions.cs

```diff
@@ -0,0 +1,19 @@
+// Assembly 'Microsoft.AspNetCore.Telemetry.Middleware'
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.AspNetCore.Telemetry;
+
+public static class HttpLoggingServiceExtensions
+{
+    public static IServiceCollection AddHttpLogging(this IServiceCollection services);
+    public static IServiceCollection AddHttpLogging(this IServiceCollection services, Action<LoggingOptions> configure);
+    public static IServiceCollection AddHttpLogging(this IServiceCollection services, IConfigurationSection section);
```

Comment:
Do we still need to ship these @Tratcher ? What's the overlap with the built in middleware after teh changes?

Reply:
See https://github.com/dotnet/extensions/pull/4330, many are removed, some are refactored.